test/cql-pytest/run.py

```diff
@@ -321,7 +321,7 @@ def run_scylla_cmd(pid, dir):
         '--strict-allow-filtering', 'true',
         '--strict-is-not-null-in-views', 'true',
         '--permissions-update-interval-in-ms', '100',
-        '--permissions-validity-in-ms', '100',
+        '--permissions-validity-in-ms', '5',
```

Comment:
From the commit message:

> ... always need to sleep for more than 100ms...

An idea you probably thought about already: introduce a loop that checks the permissions every 5 ms (or less) for no longer than 5 ms or 100 ms, so that the test can run faster but will still fail after 5 or 100 ms as previously.

Reply:
Did this (more or less) in the new version.

I still want to change the default in test/alternator/run to 5ms to make the test run faster (as I noted above, it makes this whole new test file, 27 tests and 1000 lines of code, pass in 2 seconds instead of 7 seconds), but because of the retries, the test doesn't rely on it and it can be 100ms or 10 seconds if you don't mind the test being slow.

Reply:
I hope this parameter goes away soon @nuivall . But meanwhile, is it necessary to change it for cql-pytest, perhaps sufficient to patch the alternator only?
Since this parameter will hopefully be ignored in a month or two I admit this is not very important.